client/src/global-styles/GlobalStyles.tsx

```diff
@@ -7,7 +7,7 @@ const GlobalStyles = createGlobalStyle`
    margin: 0;
 }
 body {
-   font-family: 'Roboto', sans-serif;
+   font-family: 'Poppisn', sans-serif;
```

Comment:
The font that displays is not Popins. You need to add this font eg from google fonts